content/operate/oss_and_stack/stack-with-enterprise/modules-lifecycle.md

```diff
@@ -23,11 +23,28 @@ The format is “Major1.Major2.Minor”.
 
 - The _Minor_ section of the version number represents quality improvements and fixes to existing capabilities.  The minor release number is increased when release quality improves.
 
-## Module end-of-life schedule {#modules-endoflife-schedule}
-
 End-of-Life for a given Major version is 18 months after the formal release of
 that version or 12 months after the release of the next subsequent (following) version, whichever comes last.
 
+## Module end-of-life schedule after Redis CE 8.0
+
+In Redis CE 8.0 and beyond, module version numbers will align exactly with the Redis CE versions that include them.
+For example, the modules included with Redis CE 8.0 will all have version number 8.0.
+
+This also means that Redis CE versions 8.0+ require modules with an exact match in version number.
```

Comment:
We can remove this sentence because CE does not "require" modules. It contains modules.